index.html

```diff
@@ -0,0 +1,560 @@
+<!--                        <div>-->
+<!--                            <img class="moreProjectDetailImg" src="pexels-picjumbocom-196645.jpg">-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div>-->
+<!--                        <div>-->
+<!--                            <img class="moreProjectDetailImg" src="pexels-picjumbocom-196645.jpg">-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div>-->
+<!--                        <div class="moreProjectsDetailHeading">-->
+<!--                            <h6>Description</h6>-->
+<!--                            <p>-->
+<!--                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus dolorem ea, fugit laudantium modi necessitatibus nesciunt omnis? Ab aspernatur et, illum iste quaerat quam quasi sint sit vel voluptatum? Adipisci autem blanditiis dignissimos dolor eligendi enim exercitationem facere fugit incidunt molestiae neque, odio repellat repellendus tenetur ut veniam voluptatum! Est!-->
+<!--                            </p>-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div>-->
+<!--                        <div>-->
+<!--                            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam dicta eligendi exercitationem omnis perspiciatis quod quos recusandae. Adipisci aliquid, dolorum ducimus earum magnam minima modi, obcaecati odit omnis ratione temporibus velit veritatis, voluptatibus. Alias asperiores corporis dolorum ea eligendi esse explicabo itaque, nisi nostrum, obcaecati veritatis voluptatem. Ad animi, ratione.-->
+
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div>-->
+<!--                        <div class="moreProjectsDetailHeading">-->
+<!--                            <h6>Technologies</h6>-->
+<!--                            <p>-->
+<!--                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis cum exercitationem incidunt libero officia quaerat veritatis voluptatibus? Architecto consectetur culpa cum, deleniti dolor dolorum et excepturi fuga, incidunt ipsa itaque iusto molestias, natus nobis nulla pariatur porro possimus quaerat quisquam repellendus rerum sed sequi sit soluta tempore ut vel voluptatibus.-->
+<!--                            </p>-->
+
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                </div>-->
+<!--                <div class="moreProjectsContainer">-->
+<!--                    <img class="moreProjectsImg" src="pexels-picjumbocom-196645.jpg">-->
+<!--                    <div class="moreProjectsTitle"><h5>Javascript Calculator</h5></div>-->
+<!--                    <button class="moreProjectsBtn" id="moreProjectsExpandBtn3">expand</button>-->
+<!--                    <div class="expandIcon">-->
```

Comment:
Try to remove any commented out code once you've decided you aren't going to use it any more.